deployment/v3/external/landing-page/install.sh

```
@@ -106,42 +106,47 @@ function landing_page() {
-  --set landing.smtp=$SMTP \
-  --set landing.healthservices=$HEALTHSERVICES \
-  --set landing.injiweb=$INJIWEB \
-  --set landing.injiverify=$INJIVERIFY \
-  --set istio.host=$DOMAIN \
-  --set indexFile="$indexfile"
-  
+  select_index_template
+
+  # Build helm args array
+  helm_args=(
+    -n $NS install landing-page mosip/landing-page
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)